### ISSUE TITLE
phantompeakqualtools missing from install_dependencies

### ISSUE THREAD

**Issue by jloupe** [kundajelab/chipseq_pipeline#29]
When installing dependencies, it reports an error on line 63 for cloning phantompeakqualtools.
When attempting to run the pipeline, it errors at xcor with an stderr message of no run_spp.R in directories.

**Comment by leepc12** [comment]
I tested `git clone https://github.com/kundajelab/phantompeakqualtools` but it works now.
I think this should be a temporary problem of github server?
Please uninstall dependencies and install them again.

**Comment by jloupe** [comment]
Thank you for the quick response.
We tried it a couple of times across different days, including this morning. I am not sure how it could be the server.
Since the error states run_spp.R is not in any directory, I manually added it to the chipseq_pipeline-master directory. I did a quick run starting with tagAligns and stopping at xcor, and it appears to have run that pipeline correctly. I am repeating the experiment through IDR currently to see if this fixes it.
